CryptoTechReminderSystem/UseCase/RemindLateDevelopers.cs

```diff
@@ -40,5 +32,15 @@ public void Execute(RemindLateDevelopersRequest remindLateDevelopersRequest)
                 }
             }
         }
+        
+        private bool IsHalfHourInterval()
+        {
+            return _clock.Now().ToUnixTimeSeconds() % 1800 == 0;
```

Comment:
```suggestion
            return _clock.Now().Minute % 30 == 0;
```

Reply:
I disagree good sir!

Reply:
🥇

Reply:
Resolved in https://github.com/chubberlisk/crypto_tech/pull/35/commits/001e1c2bf7a516378e65ab0e5a29b143b5e2fec6.